administration/src/application/components/ApplyController.tsx

```diff
@@ -5,30 +5,35 @@ import '@fontsource/roboto/700.css'
 
 import { useAddBlueEakApplicationMutation } from '../../generated/graphql'
 import { DialogActions } from '@mui/material'
-import useLocallyStoredState from '../useLocallyStoredState'
+import useVersionedLocallyStoredState from '../useVersionedLocallyStoredState'
 import DiscardAllInputsButton from './DiscardAllInputsButton'
 import { useGarbageCollectArrayBuffers, useInitializeGlobalArrayBuffersManager } from '../globalArrayBuffersManager'
 import ApplicationForm from './forms/ApplicationForm'
 import { useCallback, useMemo } from 'react'
 import { SnackbarProvider, useSnackbar } from 'notistack'
+import ApplicationErrorBoundary from '../ApplicationErrorBoundary'
 
-const applicationStorageKey = 'applicationState'
+const lastCommitForApplicationForm = process.env.REACT_APP_APPLICATION_COMMIT as string
```

Comment:
Maybe add a comment here where the variable is coming from, and that a preprocessor is string-replacing the process.env. stuff.